core-web/libs/dotcms-models/src/lib/dot-action-menu-item.model.ts

```diff
@@ -1,6 +1,15 @@
-import { MenuItem } from 'primeng/api';
+import { MenuItem, MenuItemCommandEvent } from 'primeng/api';
 
-export interface DotActionMenuItem {
+export interface CustomMenuItem<T = unknown> extends Omit<MenuItem, 'command'> {
+    command?(event?: T): void;
+}
```

Comment:
We remove command with the Omit and then added back? we need to change the type in the function?

Reply:
In a few cases, we call a command without parameter, and the lint fails.